contributing.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to Shadcn Organization
+
+Thank you for considering contributing to the Shadcn organization! Before you start contributing, please take a moment to read through this guide to understand how you can contribute effectively.
+
+## Introduction
+
+Welcome to Shadcn! We are a community focused on creating re-usable components using Radix UI and Tailwind CSS. However, it's important to clarify that Shadcn is not a traditional component library that you install as a dependency via npm. Instead, it's a collection of re-usable components that you can freely copy and paste into your projects.
```

Comment:
You have mentioned about `npm` package, whereas `shadcn-ui` can be installed from many different package managers (like, pnpm, yarn, bun).